### PR TITLE
update readme with CLI usage and local Webhook documentation

### PR DESCRIPTION
**Pull request by michaelangeloio**
Hi team at Stripe!

Using this command from the [firebase docs](https://firebase.google.com/products/extensions/stripe-firestore-stripe-payments) does not seem to work for me:
```
firebase ext:install stripe/firestore-stripe-payments --local --project=projectId_or_alias
```

I wanted to open this because it caused some headaches until I found the error has to do with the version ref not being supplied with that command:  https://github.com/stripe/stripe-firebase-extensions/issues/394

I presume this would be a temporary solution, but should hopefully give other devs more info when trying to set up things via the CLI. The firebase CLI should theoretically be able to pull the version in the payload shown when running the command in debug mode:

```
,"spec":{"specVersion":"v1beta","name":"firestore-stripe-payments","version":"0.3.1"
```
However, this seems like a FB CLI issue, not exactly a stripe issue.

### CODE REVIEW

File: firestore-stripe-payments/README.md [hunk 1]
```diff
@@ -50,6 +50,46 @@ Then, in the [Stripe Dashboard](https://dashboard.stripe.com):
 
 - Create a new [restricted key](https://stripe.com/docs/keys#limit-access) with write access for the "Customers", "Checkout Sessions" and "Customer portal" resources, and read-only access for the "Subscriptions" and "Prices" resources.
 
+#### Installing via Firebase CLI 
+
+When installing via the CLI, be sure to pin the version. 
+
+```
+firebase ext:install stripe/firestore-stripe-payments@0.3.1 --project=projectId_or_alias
```
Comment (dackers86) [inline firestore-stripe-payments/README.md:58]: Is there a reason for pinning to a specific version of the cli?

File: firestore-stripe-payments/README.md [hunk 2]
```diff
@@ -50,6 +50,46 @@ Then, in the [Stripe Dashboard](https://dashboard.stripe.com):
...
+```
+firebase ext:configure firestore-stripe-payments --local
+```
+
+Be sure to configure your test mode [API Key](https://stripe.com/docs/keys) and webhook [signing secret](https://stripe.com/docs/webhooks/signatures#:~:text=Before%20you%20can%20verify%20signatures,secret%20key%20for%20each%20endpoint.) when prompted. 
+
+Start the firebase emulator with:
+
+```
+firebase emulators:start
```
Comment (dackers86) [inline firestore-stripe-payments/README.md:76]:
```suggestion
firebase emulators:start --project=projectId_or_alias
```

File: firestore-stripe-payments/README.md [hunk 3]
```diff
@@ -50,6 +50,46 @@ Then, in the [Stripe Dashboard](https://dashboard.stripe.com):
...
+```
+
+Find the functions path associated with the stripe extension, typically it looks like this:
+
+- `http://localhost:3999/{projectId}/{region}/ext-firestore-stripe-payments-handleWebhookEvents`
```
Comment (dackers86) [inline firestore-stripe-payments/README.md:81]: Is there a reason for using 3999 as the function port in this example.

Typically, Firestore function ports are default as `5001`.
Another suggestion, the latest updates the Firebase cli cannot resolve `localhost`, recommending 127.0.0.1 `instead`

```suggestion
- `http://192.0.0.1:5001/{projectId}/{region}/ext-firestore-stripe-payments-handleWebhookEvents`
```

File: firestore-stripe-payments/README.md [hunk 4]
```diff
@@ -50,6 +50,46 @@ Then, in the [Stripe Dashboard](https://dashboard.stripe.com):
...
+
+- You can tunnel your local endpoint using a tool like [ngrok](https://ngrok.com/). In this case you will tunnel the localhost domain and port `http://localhost:3999`. Replace `localhost:3999` with your tunnel url. The end result would look something like:
```
Comment (dackers86) [inline firestore-stripe-payments/README.md:83]:
```suggestion
- You can tunnel your local endpoint using a tool like [ngrok](https://ngrok.com/). In this case you will tunnel the localhost domain and port `http://127.0.01:5001`. Replace `127.0.0.1:5001` with your tunnel url. The end result would look something like:
```

File: firestore-stripe-payments/README.md (hunk 1 above)
Comment (michaelangeloio) [reply firestore-stripe-payments/README.md:58]: This was so long ago I honestly forget. Feel free to suggest a change!

Comment (dackers86) [reply firestore-stripe-payments/README.md:58]: Ok, thanks!

```suggestion
firebase ext:install invertase/firestore-stripe-payments --project=projectId_or_alias

Alternatively for local source:

firebase ext:install . --project=projectId_or_alias
```

**Pull request merged**